maintainers/maintainer-list.nix

```diff
@@ -58,6 +58,9 @@
          nix-build lib/tests/maintainers.nix
 
     See `./scripts/check-maintainer-github-handles.sh` for an example on how to work with this data.
+
+    When creating a commit:
+      * When you add a maintainer: Use `maintainers: handle` as commit message.
```

Comment:
```suggestion
      * When you add yourself as a maintainer, please be aware of the commit message conventions: https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#commit-conventions
```

Reply:
In principle it can be any maintainer, not only `self`.